### PR TITLE
Update video formats

### LINKED ISSUE #1072: Improve video formats
1. Videos being located in the `static` folder make them not show up in PR previews because the previews attach the PR branch name to the beginning of the URL. See if we can put videos in a different folder(s) instead so they work consistently in local and PR previews.

2. Also, some of the embedded YouTube videos don't fit on screen on mobile devices. For example, the Unity SDK demo:
  
    image
  
    Maybe use `YoutubeEmbed` that is used in the Snaps docs to embed these videos responsively.
    We should also enable users to play the videos full-screen without opening a new a tab.

3. Increase border radius for all videos and gifs, to match our image and code sample styles.

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
# Description

- Updates embedded YouTube videos so they fit on screen on mobile devices and Unity SDK vid renders correctly
- Increases border radius for all videos and gifs, to match our image and code sample styles.

## Issue(s) fixed

Partially fixes #1072 

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

**Review by alexandratran — APPROVED** [review]
Approving for now because it worked on the previous PR, but there is no PR preview available for this PR because you pushed to a fork and not the original repo (due to how this repo is set up with GH pages). So I cannot preview these specific changes easily.

**Comment by Montoya** [comment]
I tried running this locally and could not get it to run. Seems like there may be an error. Also, I think there is a linting error too.

File: wallet/how-to/use-sdk/gaming/unity/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ tags:
   - Unity SDK
 ---
 
-import ReactPlayer from 'react-player/lazy'
+import YoutubeEmbed from '@site/src/components/YoutubeEmbed';
```
Comment (joaniefromtheblock) [inline wallet/how-to/use-sdk/gaming/unity/index.md:9]:
```suggestion
import YoutubeEmbed from '@site/src/components/YoutubeEmbed';
```

File: wallet/how-to/use-sdk/gaming/unity/index.md [hunk 2]
```diff
@@ -19,9 +19,7 @@ The MetaMask Unity SDK supports macOS, Windows, Linux, iOS, Android, and WebGL.
 
 The following video demonstrates how to install and use the MetaMask Unity SDK.
 
-<p align="center">
-  <ReactPlayer url='https://www.youtube.com/embed/0D1cIH-PZtI' />
-</p>
+<YoutubeEmbed url='https://www.youtube.com/embed/0D1cIH-PZtI' />
```
Comment (joaniefromtheblock) [inline wallet/how-to/use-sdk/gaming/unity/index.md:22]:
```suggestion
<YoutubeEmbed url='https://www.youtube.com/embed/0D1cIH-PZtI' />
```

**Review by Montoya — APPROVED** [review]

**Review by subhamkumarr — APPROVED** [review]

**Comment by joaniefromtheblock** [comment]
> I tried running this locally and could not get it to run. Seems like there may be an error. Also, I think there is a linting error.

Chatted with @Montoya about this async. Videos seem to be [rendering for me locally](https://consensys.slack.com/archives/D06LC9159HQ/p1712943417259799)

**Pull request merged**